### PR TITLE
MB-1464: Custom events

### PR DESCRIPTION
**Pull request by rlepinski**

### CODE REVIEW

File: Assets/UrbanAirship/Platforms/UAirship.cs [hunk 1]
```diff
@@ -31,6 +31,109 @@ public IEnumerable<T> AsEnumerable ()
...
+		    public double doubleValue;
+		    public bool boolValue;
+			public string[] stringArrayValue;
+
+			public Property(string type, string name, System.Object value)
+			{
+				this.type = type;
+				this.name = name;
+
+				if (type == "s") {
```
Comment (rlepinski) [inline Assets/UrbanAirship/Platforms/UAirship.cs]: should we use instance of ?

File: android-plugin-lib/src/main/java/com/urbanairship/unityplugin/UnityPlugin.java [hunk 2]
```diff
@@ -161,6 +167,84 @@ public boolean isBackgroundLocationEnabled() {
...
+                            }
+                        }
+
+                        eventBuilder.addProperty(name, strings);
+                        break;
+                    default:
+                        continue;
+                }
+            }
+        }
```
Comment (hcrowell) [inline android-plugin-lib/src/main/java/com/urbanairship/unityplugin/UnityPlugin.java]: addEvent?

Comment (rlepinski) [inline android-plugin-lib/src/main/java/com/urbanairship/unityplugin/UnityPlugin.java]: that would be helpful

**Comment by hcrowell** [comment]
:tangerine:

**Pull request merged**